src/boot/efi/boot.c

```diff
@@ -85,6 +85,7 @@ typedef struct {
         BOOLEAN editor;
         BOOLEAN auto_entries;
         BOOLEAN auto_firmware;
+        BOOLEAN bitlocker;
```

Comment:
This (and the corresponding config) could be more descriptive I think - maybe something like bitlocker_boot_via_reset? A bit longer but more self-descriptive

Reply:
Oh, right, I made a similar comment above :)